includes/class-sensei-modules.php

```diff
@@ -46,9 +46,13 @@ public function __construct( $file ) {
 		add_action( 'sensei_user_lesson_reset', array( $this, 'save_lesson_module_progress' ), 10, 2 );
 		add_action( 'wp', array( $this, 'save_module_progress' ), 10 );
 
+		add_action( 'admin_menu', array( $this, 'register_modules_admin_menu_items' ), 30 );
 		add_action( 'admin_post_order_modules', array( $this, 'handle_order_modules' ) );
+
 		add_filter( 'manage_course_posts_columns', array( $this, 'course_columns' ), 11, 1 );
 		add_action( 'manage_course_posts_custom_column', array( $this, 'course_column_content' ), 11, 2 );
+		add_filter( 'manage_lesson_posts_columns', array( $this, 'lesson_columns' ), 11, 1 );
```

Comment:
I'll leave the same feedback here as on another PR about preferring function names to start with a verb, so `add_lesson_columns`, `add_lesson_column_content` or similar. I know the course ones are `course_columns` and `course_column_content`, but they're poorly named IMO. 🙂

Reply:
Yeah, I should stop using the older code as an example. Updated in c561ce2c37c3f18269a0459a3f51a52817950d08.